packages/app/src/components/MenuBar/MenuBar.js

```diff
@@ -32,7 +32,8 @@ const onNew = () => {
     }
 }
 const getOnRename = props => details => props.onRenameVisualization(details)
-const getOnSave = props => details => props.onSaveVisualization(details, false)
+const getOnSave = props => (details = {}) =>
+    props.onSaveVisualization(details, false)
```

Comment:
Why not leave the the default prop handling to `onSaveVisualization` instead of using a `details = {}` here?

Reply:
Sorted.